src/test/java/HomePage.java

```diff
@@ -0,0 +1,27 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class HomePage extends BasePage{
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+
+    By playlistField = By.cssSelector("#playlists li:nth-child(3)");
+    By newPlaylist = By.cssSelector("[name='name']");
+    By notification = By.cssSelector("div.success.show");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```